governance.md

```diff
@@ -1,149 +1,285 @@
+    - if they choose public membership, they get a badge on their github profile
+    - TBD: swag
+- **REVIEWER**:
+  - includes all requirements, expectations, and benefits of MEMBERS
+  - Requirements
+    - org member for at least 3 months
+    - authored and/or reviewed at least 20 merged non-trivial PRs, including at least 3 as the primary
+      reviewer
+    - familiar enough with some part of the codebase to be in an [OWNERS](contributors/devel/owners.md)
+      file `reviewers` list
+    - nominated by an APPROVER for that part of the codebase, with no objections from other
+      APPROVERS
+    - may be asked to become a REVIEWER as a precondition for accepting a large code contribution
+  - Expectations
+    - responsible for project quality control
+    - will be assigned related PRs and test bugs
+    - responsive to [code review](contributors/devel/collab.md) requests as per 
+      [community expectations](contributors/devel/community-expectations.md)
+  - Benefits
+    - can champion incubator repos
+    - added to the repo’s reviewers github team (e.g., [`kubernetes-reviewers`](https://github.com/orgs/kubernetes/teams/kubernetes-reviewers))
+    - gets "read access" to the repository
+    - gets a badge on PR and issue comments
+    - TBD: swag (e.g., ribbons at conferences, tshirts)
+- **APPROVER**:
+  - includes all requirements, expectations, and benefits of REVIEWERS
+  - Requirements
+    - previously a REVIEWER for that part of the codebase for at least 3 months
+    - authored and/or reviewed at least 30 merged non-trivial PRs, including at least 10 as the primary
+      reviewer
+    - familiar enough with some part of the codebase to be in an [OWNERS](contributors/devel/owners.md)
+      file `approvers` list
+    - demonstrated sound technical judgement
+    - nominated by a relevant APPROVER, with no objections from other APPROVERS in that area
+    - may be asked to become an APPROVER as a precondition for accepting a large code contribution
+  - Expectations
+    - responsive to [code review](contributors/devel/collab.md) requests as per
+      [community expectations](contributors/devel/community-expectations.md)
+    - mentor REVIEWERS and CONTRIBUTORS
+  - Benefits
+    - TBD: swag
+- **MAINTAINER**:
+  - includes all requirements, expectations, and benefits of APPROVERS
+  - Requirements
+    - APPROVER for some part of the codebase for at least 3 months
+    - on project for at least 1 year
+    - authored and/or reviewed at least 50 merged non-trivial PRs, including at least 20 as the primary
+      reviewer
+    - familiar enough with the codebase and project conventions to be in the repo’s top-level OWNERS
+      file `maintainers` list
+    - committed to the project: Kubernetes is a very high-volume project, with hundreds of PRs and issues
+      per week, so expect a significant time commitment 
+    - nominated by a Champion (see below) from the existing MAINTAINERS, who will find a
```

Comment:
Yes, it's new, but was iterated on in a prior PR and then again in the doc.

The maintainers group is currently only represented in a github team. The problem with that is that many people can't see who is in that team and almost nobody can observe changes to the team.

The set of maintainers has been frozen since last summer pending some kind of official process.